GeometryService/src/ExtShieldUpstreamMaker.cc

```diff
@@ -139,7 +139,14 @@ namespace mu2e {
 
       } // end loop over boxes of type...
     } // end loop over types...
-
+    //------------------Remote Handling Door-------------------------
+    // get DoorParameterA and DoorParameterB from simple config.
+    const double DParamA = c.getDouble("ExtShieldUpstream.DoorParameterA");
```

Comment:
Yes, DoorParameterA: the amount by which the door is open or fully closed.
DoorParameterB: Away from the wall.
I can change the names in the simple config file, is that okay?
Do you have any suggestions for the names?

Reply:
You need to change the names in both the simple config file and in these code that reads them.   The names need to match.  Since you just introduced them in this PR the change is local.  Or were you asking something else?

For names, something like: 
FrameGap
OpenDistance